src/bot/cogs/commands.py

```
@@ -148,6 +149,17 @@ async def execute_actions(self, message: discord.Message, cmd_data: dict, args:
                 print(f"Action {action_type} failed: {e}")
                 continue
 
+    async def process_admin_cmd(self, message: discord.Message, name: str, args: list[str]):
+        if not str(message.author.id) in ADMINS:
```

Ruff (0.15.7)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)